ceph.spec.in

```diff
@@ -908,6 +908,17 @@ depending on python-ceph should be fixed to depend on python-rados,
 python-rbd, python-rgw or python-cephfs instead.
 %endif
 
+%package grafana-dashboards
+Summary:	The set of grafana dashboards for monitoring purposes
+%if 0%{?suse_version}
+Group:		System/Filesystems
+%endif
+%description grafana-dashboards
+This package provides a set of grafana dashboards for monitoring of
+ceph clusters. The dashboards require a Prometheus server setup with
+ceph-mgr prometheus collector and node_exporter collectors. These
```

Comment:
Prometheus with capital P

Reply:
Here, I was referring to the prometheus collector/exporter ceph-mgr module (just like the node_exporter module/package). I suppose I should either capitalize both (Prometheus and Node Exporter) or keep them as they are?

Reply:
@b-ranto Ah, in that case, then drop the first instance of the word "collector" (since you have "collectors" at the end of the sentence). So it would read like this:

"The dashboards require a Prometheus server setup with the ceph-mgr "prometheus" and "node_exporter" collectors."

?

Reply:
(I like to put non-standard usage in double-quotes to call the reader's attention to the fact that it's a special case.)